.github/workflows/release-prod.yml

```diff
@@ -124,3 +124,28 @@ jobs:
         curl -X POST -H "Content-Type:application/json" \
         -d "{\"version\":\"$BASE_VERSION\",\"date\":\"$DATE\"}" \
         $RELEASE_TRACKER_URL/logProduction
+    - name: Create Github release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        # Get the newest release tag for the firebase package (e.g. firebase@10.12.0)
+        NEWEST_TAG=$(git describe --tags --match "firebase@[0-9]*.[0-9]*.[0-9]*" --abbrev=0)
+
+        # Get the release notes from the description of the most recent merged PR into the "release" branch
+        # See: https://github.com/firebase/firebase-js-sdk/pull/8236 for an example description
+        RELEASE_NOTES=$(gh pr list \
```

Comment:
Were we going to add in some text to say "For more detailed release notes, see [Firebase JavaScript SDK Release Notes](https://firebase.google.com/support/release-notes/js)."? You might want to test if you have to escape any of those markdown linking characters.

Changesets are better than commits but I think release notes may have more info than changesets.

Reply:
Done. Here's an example release with the new release notes: https://github.com/dlarocque/firebase-js-sdk/releases/tag/firebase%405.5.5